querydsl-codegen/src/test/java/com/querydsl/codegen/GenericExporterTest.java

```diff
@@ -117,5 +117,11 @@ public void Export_Domain_Package() {
         exporter.export(Cat.class.getPackage());
     }
 
+    @Test
+    public void Export_SerializerConfig() {
+        exporter.setTargetFolder(new File("target/gen7"));
+        exporter.setSerializerConfig(new SimpleSerializerConfig(true, true, true, true, ""));
+        exporter.export(Cat.class.getPackage());
```

Comment:
At least, since it's `codegen`, if the `code` is `gen`-ed it's a step in the right direction.
But assertions on its content is also good.